assets/javascripts/kitten/components/organisms/header-menu/styles.js

```diff
@@ -12,11 +12,26 @@ export const StyledList = styled.ul`
   border: none;
   list-style: none;
 
+  &.k-HeaderMenu--left {
+    .k-HeaderMenu__item {
+      .k-HeaderMenu__item__link {
+        text-align: left;
+      }
+    }
+  }
+
+  &.k-HeaderMenu--right {
+    .k-HeaderMenu__item {
+      .k-HeaderMenu__item__link {
+        text-align: right;
+      }
+    }
+  }
+
```

Comment:
ça serait peut-être plus simple comme ça

```suggestion
  &.k-HeaderMenu--left .k-HeaderMenu__item .k-HeaderMenu__item__link {
    text-align: left;
  }

  &.k-HeaderMenu--right .k-HeaderMenu__item .k-HeaderMenu__item__link {
    text-align: right;
  }

```